frontend/src/graphql-config.js

```diff
@@ -2,7 +2,7 @@ import { ApolloClient, InMemoryCache } from "@apollo/client";
 const uri =
   process.env.NODE_ENV === "production"
     ? "https://csc-309-geoart.herokuapp.com/graphql?"
-    : "http://0.0.0.0:8080/graphql?";
+    : "http://localhost:8080/graphql?";
```

Comment:
Yeah sorry about that. The four zeros were just a design choice, but it probably has some restricted meaning

Reply:
We live and we learn!